in_toto/models/metadata.py

```diff
@@ -216,9 +422,9 @@ def verify_signature(self, verification_key):
     Raises:
       securesystemslib.exceptions.FormatError: The passed key is malformed.
 
-      SignatureVerificationError: No signature keyid matches the verification
-          key keyid, or the matching signature is malformed, or the matching
-          signature is invalid.
+      securesystemslib.exceptions.VerificationError: No signature keyid matches
```

Comment:
I think the old function should keep the old exception until we deprecate it.

Reply:
What are your thoughts on this? Again, for the sake of backwards-compatibility, we could keep `SignatureVerificationError` and, for the sake of a consistent interface, change `Envelope.verify_signature` to something like:
```python
# ...
except VerificationError:
    raise SignatureVerificationError
# ...
```